tests/cli/restart_test.go

```diff
@@ -17,8 +17,8 @@ func TestRestart(t *testing.T) {
 		},
 		{
 			"dynos -a acme-inc",
-			`v1.web.1  1X  running   5d  "./bin/web"
-v1.web.2  1X  running   5d  "./bin/web"`,
+			`v1.web.1    1X  running   5d  "./bin/web"
```

Comment:
Can we also update the [fake scheduler to add a fake host](https://github.com/remind101/empire/blob/master/scheduler/fake.go#L55) to the returned set of instances? Maybe just some string that looks like an ec2 instance id.